src/App.tsx

```diff
@@ -1,13 +1,46 @@
-import React, { useState } from 'react';
-import './App.css';
+import "./App.css";
+import { Header, HeroSection, Toast } from "./Components";
+import { BrowserRouter as Router } from "react-router-dom";
+import { useEffect } from "react";
+import axios from "axios";
+import { useStore } from "./Store";
+import URL from "./Components/ServerURL";
 
 const App = () => {
+  const { storeState, storeDispatch } = useStore();
+  const { loadingMessage } = storeState;
+
+  const fetchQuizzes = async () => {
+    storeDispatch({ type: "IS_LOADING", payload: "loading" });
+    try {
+      const response = await axios(`${URL}/quiz`);
+      const quizzes = response.data.quizzes;
+      if ((response.status = 200)) {
+        storeDispatch({ type: "LOAD_QUIZZES", payload: quizzes });
```

Comment:
we can destructure the response  `const {data:{quizzes},status`

Reply:
Done.